library/core/src/intrinsics.rs

```diff
@@ -3503,6 +3528,28 @@ pub const unsafe fn write_bytes<T>(dst: *mut T, val: u8, count: usize) {
     }
 }
 
+/// Return whether the initialization state is preserved.
```

Comment:
I don't really understand what this means.

Reply:
This is copying bytes, whether they are initialized or not. The copies of non-initialized bytes (such as padding bytes) should result in a non-initialized copy, and it is UB to read them.

Reply:
Can you make the comment say what you gave me as an explanation?